README.md

```diff
@@ -1,7 +1,7 @@
-'gschem and Friends' Electronic Design Suite
-============================================
+Lepton Electronic Design Suite
+==============================
 
-Copyright (C) 1998-2015 gEDA Developers
+Copyright (C) 1998-2016 gEDA Developers, 2016-2017 Lepton Developers.
```

Comment:
Please add another copyright line for Lepton Developers. And since Lepton started in 2017, first copyright should be *1998-2017*, and second just *2017*